src/nav/AppWithNavigationState.js

```diff
@@ -36,13 +37,16 @@ class AppWithNavigation extends PureComponent<Props> {
     const addListener = createReduxBoundAddListener('root');
 
     return (
-      <AppNavigator
-        navigation={addNavigationHelpers({
-          state: nav,
-          dispatch,
-          addListener,
-        })}
-      />
+      <View style={{ flex:1 }}>
+        <ZulipStatusBar />
```

Comment:
Now if we thinks from top view: then for all those routes (screen) which are using `src/common/Screen`, have two `ZulipStatusBar`. 
One is this one and another one present in the `Screen`. 
Consider and include this case too.

Problem what I can think of this is we are changing status bar color according to stream color on narrow, chat screen is in app navigator, there can be multiple chat screen in stack but we only needs to care for top one. This will be trickiest part to handle :) 👍